src/server/src/main/java/com/spotify/reaper/resources/RepairRunResource.java

```diff
@@ -520,54 +550,54 @@ public Set splitStateParam(Optional<String> state) {
-              "Repair run with id \"" + runId
-              + "\" is currently running, and must be stopped before deleting").build();
-        }
-        if (!runToDelete.get().getOwner().equalsIgnoreCase(owner.get())) {
-          return Response.status(Response.Status.FORBIDDEN).entity(
-              "Repair run with id \"" + runId + "\" is not owned by the user you defined: "
-              + owner.get()).build();
-        }
-        if (context.storage.getSegmentAmountForRepairRunWithState(runId, RepairSegment.State.RUNNING) > 0) {
-          return Response.status(Response.Status.FORBIDDEN).entity(
-              "Repair run with id \"" + runId
-              + "\" has a running segment, which must be waited to finish before deleting").build();
-        }
-        // Need to get the RepairUnit before it's possibly deleted.
-        Optional<RepairUnit> unitPossiblyDeleted =
-            context.storage.getRepairUnit(runToDelete.get().getRepairUnitId());
-        int segmentsRepaired =
-            context.storage.getSegmentAmountForRepairRunWithState(runId, RepairSegment.State.DONE);
-        Optional<RepairRun> deletedRun = context.storage.deleteRepairRun(runId);
-        if (deletedRun.isPresent()) {
-          RepairRunStatus repairRunStatus =
-              new RepairRunStatus(deletedRun.get(), unitPossiblyDeleted.get(), segmentsRepaired);
-          return Response.ok().entity(repairRunStatus).build();
-        }
+      if (runToDelete.get().getRunState() == RepairRun.RunState.RUNNING) {
+        return Response.status(Response.Status.FORBIDDEN).entity(
+            "Repair run with id \"" + runId
+            + "\" is currently running, and must be stopped before deleting").build();
+      }
+      if (!runToDelete.get().getOwner().equalsIgnoreCase(owner.get())) {
+        return Response.status(Response.Status.FORBIDDEN).entity(
+            "Repair run with id \"" + runId + "\" is not owned by the user you defined: "
+                + owner.get()).build();
+      }
+      if (context.storage.getSegmentAmountForRepairRunWithState(runId, RepairSegment.State.RUNNING) > 0) {
+        return Response.status(Response.Status.FORBIDDEN).entity(
+            "Repair run with id \"" + runId
+            + "\" has a running segment, which must be waited to finish before deleting").build();
+      }
+      // Need to get the RepairUnit before it's possibly deleted.
+      final Optional<RepairUnit> unitPossiblyDeleted =
+          context.storage.getRepairUnit(runToDelete.get().getRepairUnitId());
+      final int segmentsRepaired =
+          context.storage.getSegmentAmountForRepairRunWithState(runId, RepairSegment.State.DONE);
+      final Optional<RepairRun> deletedRun = context.storage.deleteRepairRun(runId);
+      if (deletedRun.isPresent()) {
+        final RepairRunStatus repairRunStatus =
+            new RepairRunStatus(deletedRun.get(), unitPossiblyDeleted.get(), segmentsRepaired);
```

Comment:
nit: this statement, and those on 579 and 577, can wrap with the `=` starting the new line. ref https://github.com/thelastpickle/cassandra-reaper/commit/99aacd057693dee9967bf056e2dbf37d48a03930#commitcomment-23993308